src/api/window/window.ts

```diff
@@ -953,11 +973,12 @@ export class _Window extends EmitterBase<WindowEvents> {
     /**
      * Sets the window's size and position.
      * @property { Bounds } bounds This is a * @type {string} name - name of the window.object that holds the propertys of
+     * @param { WindowMovementOptions } options Optional parameters to modify window movement
      * @return {Promise.<void>}
      * @tutorial Window.setBounds
      */
-    public setBounds(bounds: Bounds): Promise<void> {
-        return this.wire.sendAction('set-window-bounds', Object.assign({}, this.identity, bounds)).then(() => undefined);
+    public setBounds(bounds: Bounds, options: WindowMovementOptions = { moveIndependently: false }): Promise<void> {
+        return this.wire.sendAction('set-window-bounds', Object.assign({}, this.identity, { bounds, options })).then(() => undefined);
```

Comment:
keep the bounds property top level to ensure backwards compatibility

Reply:
Ahhh duh, reverted, thanks!